images/runtime/node/8.8/Dockerfile

```diff
@@ -81,8 +81,8 @@ CMD [ "node" ]
 
 FROM node-original-image as oryx
 
-RUN  if command -v apt; then \
-       apt update && apt upgrade --yes; \
+RUN  if command -v apt-get; then \
```

Comment:
Do we really need this `command` test? Isn't this image **always** debian-based?

Reply:
I do not know actually. Let's fix it in a separate PR if this we really care about it.